lib/tag-finder.coffee

```diff
@@ -8,8 +8,8 @@ SelfClosingTags = require './self-closing-tags'
 module.exports =
 class TagFinder
   constructor: (@editor) ->
-    @tagPattern = /(<(\/?))([^\s>]+)([^/>]*)(\/?)(>)/
-    @wordRegex = /[^>\r\n]*>/
+    @tagPattern = /(<(\/?))([^/\s>]+)([^/>]*)(\/?)(>)/
+    @wordRegex = /(?![\n\r])[^>]*>/
```

Comment:
1. The `(?![\n\r])` CAN be changed to `(?!$)`, thank you for the suggestion
2. So what I'm trying to achieve with the negated lookahead at the start of the match was to only match the string if it was not ***preceded*** by a newline or carriage return. In other words, what looks like a negated lookahead is actually, functionally, a negated lookbehind, and is placed exactly where it needs to be to make the whole thing work. (the tests start failing if I mess with it)